src/device/CrashDumpManager.cpp

```
@@ -0,0 +1,124 @@
+#include "device/CrashDumpManager.hpp"
+
+#include <stdexcept>
+
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)